### PR TITLE
Team 3 V1 Agent 

### PR DESCRIPTION
**Pull request by Neelesh99**
# Summary

Here's our agent. We've put much sweat and tears into it, and have affectionately dubbed it Agrim.

## Test Plan

Yes

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

**Comment by preetl** [comment]
AHAHAHAHAHA 
image

**Review by preetl — APPROVED** [review]
yeet

File: internal/clients/team3/clientconfig.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	return islandParams{
+		equity:                      0.1, // 0-1
+		complianceLevel:             0.1, // 0-1
+		resourcesSkew:               1.3, // >1
+		saveCriticalIsland:          true,
+		escapeCritcaIsland:          true,
+		selfishness:                 0.3,  //0-1
+		minimumRequest:              30,   // real number
+		disasterPredictionWeighting: 0.4,  // 0-1
+		recidivism:                  0.3,  // real number
```
Comment (JamesTev) [inline internal/clients/team3/clientconfig.go:13]: recidivism 😂

**Pull request merged**